app/src/components/UserDetailsHeader.tsx

```diff
@@ -45,7 +46,10 @@ const UserDetailsHeader: React.FC = () => {
       const { fingerPrint } = key;
 
       const lastEightChar =
-        fingerPrint?.slice(fingerPrint.length - 8) ?? "Missing PGP Fingerprint";
+        fingerPrint
+          .split(":")
+          .join(" ")
+          .slice(fingerPrint.length - 10) ?? "Missing PGP Fingerprint";
```

Comment:
variable name lastEightChar slice last 10 char

Reply:
Updated name to lastEightCharAndSpaces